src/app/frontend/chrome/chrome.scss

```diff
@@ -22,6 +22,7 @@
 
 .kd-toolbar {
   box-shadow: $whiteframe-shadow-z1;
+  min-height: 64px;
```

Comment:
I've found that when width of the browser changes (is too small) then height of the toolbar changes also, so I've set it to current minimum.